docs/api-docs/headless/carts.md

```diff
@@ -28,7 +32,28 @@ X-Auth-Token: {{ACCESS_TOKEN}}
             "product_id": 80,
             "variant_id": 64
         }
-    ]
+    ],
+    "locale": "en-us"
+}
+```
+To create a cart with option selections, include `option_id` and `option_value` in your `POST` request.
+```json
```

Comment:
```suggestion
```json lineNumbers
```

Reply:
@bc-traciporter I noticed that the other code blocks don't have "lineNumbers," so if you commit my suggestion, there are others also that need to be changed

Reply:
@bc-andreadao what does lineNumbers do? Can you tell me why it should be added?

Reply:
It adds lineNumbers for your code blocks

Reply:
![Uploading linenumbers.png…]()